docs/docker.md

```diff
@@ -0,0 +1,72 @@
+# Running all the components with docker compose
+
+There are 4 docker compose files provided to run origin stack:
+
+-   based on the latest **stable** versions of NPM packages:
+    -   Vanilla Origin version with a **custom Device Registry** (`origin.release.yml`)
+    -   **I-REC compliant** Origin version (`origin.release.irec.yml`)
+-   based on the latest **canary** versions of NPM packages:
+    -   Vanilla Origin version with a **custom Device Registry** (`origin.local.yml`)
+    -   **I-REC compliant** Origin version (`origin.local.irec.yml`)
+
+All compose files require docker images to be built locally.
+
+## Building docker images locally
+
+<div class="admonition attention">
+  <p class="first admonition-title">Attention</p>
+  <p class="last">
+    Make sure you completed <a href="../getting-started">Getting started</a> in order to install and build Origin monorepo.
+  </p>
+</div>
+
+-   Check out the latest code from the [Origin repository](https://github.com/energywebfoundation/origin)
+-   Execute the following commands from the code folder to build the codebase:
+
+```shell
+rush update
+rush build #in case of errors it is necessary to execute rush rebuild
+```
+
+-   Build docker images
+
+    -   For the latest stable release:
+
+        ```shell
+        yarn build:containers:release
+        ```
+
+    -   For the latest canary version:
+
+        ```shell
+        yarn build:containers:canary
+        ```
+
+## Start chosen stack
+
+To start chosen stack, for example **I-REC compliant** Origin, execute the following:
+
+```shell
+docker-compose -f origin.release.irec.yml up -d
```

Comment:
Please use `origin.local.yml ` as the example YAML file in this file, as that one will probably be the only one that new devs will use. `origin.release.irec.yml` will only ever be used by the CI/CD.